static/js/click_handlers.js

```diff
@@ -430,6 +430,11 @@ exports.initialize = function () {
         recent_topics.update_filters_view();
     });
 
+    $("body").on("click", "td.recent_topic_stream", (e) => {
+        e.stopPropagation();
+        window.location.href = $(e.currentTarget).find("a").attr("href");
+    });
+
```

Comment:
Can you split out this change in a separate commit? Other than that, this PR looks good to me.